test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1151,9 +1153,11 @@ func makeNodeUnschedulable(c clientset.Interface, node *v1.Node) error {
 	return fmt.Errorf("Failed to taint node in allowed number of retries")
 }
 
+// CriticalAddonsOnlyError implements the `error` interface, and signifies the
+// presence of the `CriticalAddonsOnly` taint on the node.
 type CriticalAddonsOnlyError struct{}
 
-func (_ CriticalAddonsOnlyError) Error() string {
+func (c CriticalAddonsOnlyError) Error() string {
```

Comment:
Is there any reason for this change? We don't use the object in this method

Reply:
Great question! The golinter was complaining about the use of `_`, so I presumed the idiomatic go way was to use a named variable. But definitely happy to undo this change. Just let me know.

Reply:
That's weird, freshly updated golint doesn't complain. And in Go libraries, this style is used:
func (CriticalAddonsOnlyError)

Reply:
Cool, I will switch to that still. Update coming shortly.

Reply:
@aleksandra-malinowska fixed! Ready for another look whenever you get the chance.